python/GafferUITest/NodeGraphTest.py

```diff
@@ -96,5 +96,32 @@ def testTitle( self ) :
 
 		self.assertEqual( g.getTitle(), "This is a test!" )
 
+	def testArrangeNodes( self ) :
+
+		s1 = Gaffer.ScriptNode()
+		s2 = Gaffer.ScriptNode()
+
+		s1["n1"] = GafferTest.AddNode()
+		s1["n2"] = GafferTest.AddNode()
+
+		s2["n1"] = GafferTest.AddNode()
+		s2["n1"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = IECore.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
```

Comment:
Again, we shouldn't be accessing this directly. This is what `GraphGadget.setNodePosition()` is for.